### ISSUE TITLE
Regression error problem

### ISSUE THREAD

**Issue by bilalkah** [zylo117/Yet-Another-EfficientDet-Pytorch#667]
I try to train efficientDet-D0 pre-trained with custom dataset. Dataset name is uyz so I create a project file called uyz.yml in projects folder. I copied .yml file from logo.yml and changed project name and obj_list. When I train the model I get 0 Reg Loss. What do I wrong. I suspect the problem is anchor boxes. How can I calculate new scales and ratios. Also, should I change mean and std for my dataset? Or is there any problem which I did not see?
![image](https://user-images.githubusercontent.com/45990633/124465775-786e5500-dd9e-11eb-9683-6a22a156d21d.png)
Here is my custom dataset: https://drive.google.com/drive/folders/19FsxgEGOsgVnH50C2B2ay0b1RYXsgbnv?usp=sharing

Here is the uyz.yml. I changed anchors with some value to try.
```
project_name: uyz  # also the folder name of the dataset that under data_path folder
train_set: train
val_set: val
num_gpus: 1

# mean and std in RGB order, actually this part should remain unchanged as long as your dataset is similar to coco.
mean: [ 0.485, 0.456, 0.406 ]
std: [ 0.229, 0.224, 0.225 ]

# this anchor is adapted to the dataset
anchors_scales: '[2 ** 0, 2 ** (1.0 / 3.0), 2 ** (2.0 / 3.0)]'
anchors_ratios: '[(0.7, 1.4), (1.0, 1.0), (1.2, 1.0)]'

obj_list: [ 'car', 'human']
```

**Comment by bilalkah** [comment]
The problem was category ids. Solved.
https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/issues/386#issue-637519692
Question 4
